powersimdata/scenario/create.py

```diff
@@ -236,7 +240,7 @@ def get_profile(self, kind):
                 self.get_grid(),
                 self.get_ct(),
             )
-            return profile.get_profile(kind)
+            return profile.get_profile(kind).loc[self.start_date : self.end_date]
```

Comment:
Do we still need the `.loc` here?

Reply:
Good catch: no, it's now redundant (here and in `Ready`). Fixed.